packages/jaspr_lints/lib/src/assists/tree_assist.dart

```diff
@@ -127,25 +117,15 @@ class TreeAssistProvider extends DartAssist {
     var content = resolver.source.contents.data;
 
     if (children.length == 1) {
-      final cb = reporter.createChangeBuilder(
-        priority: 3,
-        message: 'Remove this component',
-      );
-
-      cb.addDartFileEdit((builder) {
+      reporter.createChangeBuilder(priority: 3, message: 'Remove this component').addDartFileEdit((builder) {
         builder.addReplacement(node.sourceRange, (edit) {
           var child = children.first;
           var childIndent = getLineIndent(resolver.lineInfo, child);
           edit.write(content.substring(child.offset, child.end).reIndent(lineIndent - childIndent, skipFirst: true));
         });
       });
     } else if (node.parent is ListLiteral) {
-      final cb = reporter.createChangeBuilder(
-        priority: 3,
-        message: 'Remove this component',
-      );
-
-      cb.addDartFileEdit((builder) {
+      reporter.createChangeBuilder(priority: 3, message: 'Remove this component').addDartFileEdit((builder) {
```

Comment:
What do you think about this:
```
reporter.createChangeBuilder(
   priority: 3,
   message: 'Remove this component'
).addDartFileEdit((builder) {
  ...
}
```
Lines then are not so long..

Reply:
It would format that way, which isn't nice:

```dart
reporter
          .createChangeBuilder(
        priority: 2,
        message: 'Wrap with $name',
      )
          .addDartFileEdit((builder) {
```